.github/workflows/ci.yml

```diff
@@ -70,6 +70,11 @@ jobs:
         restore-keys: |
           ${{ runner.os }}-go-        
     - run: GOPROXY=direct GOSUMDB=off GO111MODULE=on go build -tags draft ./cmd/main.go
+    - name: Upload artifacts
```

Comment:
I don't think this is needed 🤔

Reply:
I explained the reasons to use it in [this comment below](https://github.com/layer5io/meshery/pull/1862/files#r518824113)